.github/workflows/main.yml

```
@@ -63,12 +65,17 @@ jobs:
       with:
         name: conda-bld
         path: conda-bld
+    - name: Setup Miniconda
+      uses: conda-incubator/setup-miniconda@a4260408e20b96e80095f42ff7f1a15b27dd94ca

```

Please keep the real version as comment... both dependabot / renovate also update the version in comment
```suggestion
      uses: conda-incubator/setup-miniconda@a4260408e20b96e80095f42ff7f1a15b27dd94ca # v3.0.4
```

---

Thanks!